src/main/resources/lang/english.lang

```diff
@@ -532,7 +529,6 @@ chat styles:
 	strikethrough: strikethrough, strike, s
 	underline: underlined, underline, u
 	magic: magic, obfuscated
-	obfuscated: magic, obfuscated
```

Comment:
Why was all of this stuff removed?

Reply:
Uhh when I merged the master branch with my branch it must've automatically removed it, I have no clue as to why or how

Reply:
I'll revert that change